CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
   * Fixed an issue for clusters older than v1.20.X where Cluster Autoscaler cannot be started or reconciled.
   * Fixed an issue where clusters using Pod Identity would fail to migrate to Azure CNI.
   * The Istio-based service mesh add-on components can now tolerate running on the [system node pools](https://learn.microsoft.com/azure/aks/use-system-pools?tabs=azure-cli#system-and-user-node-pools) with the `CriticalAddonsOnly` taint.
+  * Fixed an issue where ephemeral disk placement was incorrectly updated.
```

Comment:
Fixed an issue where the ephemeral disk placement was incorrectly modified/updated on an existing nodepool.